zerver/openapi/zulip.yaml

```diff
@@ -14080,6 +14086,14 @@ paths:
                           - 2 = Zulip Cloud free plan (LIMITED)
                           - 3 = Zulip Cloud Standard plan (STANDARD)
                           - 4 = Zulip Cloud Standard plan, sponsored for free (STANDARD_FREE)
+                      realm_enable_guest_user_indicator:
+                        type: boolean
+                        description: |
+                          Present if `realm` is present in `fetch_event_types`.
+
+                          Whether to add "(guest)" indicator to user names in UI.
+
+                          **Changes**: New in Zulip 8.0 (feature level 216).
```

Comment:
This probably deserves a bit of extra exposition; I might just add it as part of merging.